Change `securityContext` variable name to allow setting the PSP/SCC seLinux `type` or `rule` (#46)
DataDog/helm-charts: charts/datadog/templates/NOTES.txt

If you cannot upgrade to Agent 6.x or later, you can use a previous version
of the chart by calling helm install with `--version 0.18.0`.
{{- end }}

{{- if .Values.agents.podSecurity.securityContext }}

#################################################################
####               WARNING: Deprecation notice               ####
#################################################################

You are using the datadog.podSecurity.securityContext parameter, which has been renamed datadog.podSecurity.seLinuxContext.

> This version still supports datadog.podSecurity.securityContext, but it will be removed in a future version of our Helm chart.

Reviewer: ```suggestion
This version still supports datadog.podSecurity.securityContext, but it will be removed in the next major version of our Helm chart.
```
